pkg/dispatcher/dispatcher/dispatcher.go

```diff
@@ -255,8 +255,9 @@ func (d *DispatcherImpl) closeConsumerGroup(subscriber *SubscriberWrapper) {
 // If there aren't any consumer-specific differences between the current config and the new one,
 // then just log that and move on; do not restart the ConsumerGroups unnecessarily.
 func (d *DispatcherImpl) ConfigChanged(configMap *v1.ConfigMap) Dispatcher {
-	d.Logger.Debug("New ConfigMap Received", zap.String("configMap.Name", configMap.ObjectMeta.Name))
 
+	// Create A New Sarama Config (Carrying Forward The Kafka Secret Auth)
```

Comment:
Is this comment for the kafkasarama.NewSaramaConfig() line below it?  That line isn't what carries forward the auth values, it's the UpdateSaramaConfig() call later, right?

Reply:
Yeah, that was missed when I reverted back my refactor of the lifecycle so everything ran through the constructor - will remove the confusing part.